src/oc_erchef/apps/oc_chef_authz/src/oc_chef_policy.erl

```diff
@@ -181,12 +181,6 @@ record_fields(_ApiVersion) ->
 list(#oc_chef_policy{org_id = OrgId} = P, CallbackFun) ->
     CallbackFun({list_query(P), [OrgId], rows}).
 
-update(#oc_chef_policy{
-                      org_id = _OrgId,
-                      authz_id = _PolicyAuthzId,
-                      last_updated_by = _AuthzId
-                     } = Record, CallbackFun) ->
-    chef_object_default_callback:update(Record, CallbackFun).
```

Comment:
note that the module here should be chef_object_default_callbacks, not chef_object_default_callback.  Same below as well.